internal-resources/git-workflow-and-contributor-guide.adoc

```diff
@@ -4,135 +4,226 @@
 :numbered:
 :internal-guide:
 
-= Stonesoup contributor guide
+= {Product_Name} Contributor Guide
 
-== Overview
+== How to use this guide
 
-This document describes the tasks needed to contribute to stonesoup documentation.
+The Contributor Guide is for writers on the {Product_Name} team, or anyone who needs to edit our documentation.
 
-IMPORTANT: If at *ANY TIME* you feel lost or don't know what to do, start asking questions on the _#forum-hac-docs_ slack channel.
+NOTE: To get help at any point, post in the _#forum-hac-docs_ Slack channel.
 
-* Before you begin, you need to perform the following one time process to xref:install_and_configure_git[install and configure Git]
+We organized this guide into the following sections:
 
-* Once you have configured *Git* and set up your repositories, use this xref:documentation_update_process[documentation update process] to make changes to the stonesoup documentation.
+* *Getting started*: the Getting started section outlines all of the tools that you need to install before you can begin working on our documentation. The steps in this section should be a one-time process.
+* *Typical workflow for editing the documentation*: this section represents the tasks that a writer needs to do on a weekly basis to maintain the documentation. Use this section for every day guidance while you do your work. 
+* *Advanced resources*: The Advanced resources section is for edge cases, or when you find errors that you do not know how to fix. We do not update or maintain the information in this section, so use it with caution. The sections might not follow style guide principles or be up to date.
```

Comment:
Really good idea having an explanation of the different section of this guide! Really good description of each section.